_posts/en/newsletters/2024-11-22-newsletter.md

```diff
@@ -0,0 +1,145 @@
+---
+title: 'Bitcoin Optech Newsletter #330'
+permalink: /en/newsletters/2024/11/22/
+name: 2024-11-22-newsletter
+slug: 2024-11-22-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Pluggable channel factories:** ZmnSCPxj [posted][zmnscpxj plug] to
+  Delving Bitcoin a proposal to make a small set of changes to the
+  [BOLT][bolts repo] specification to allow existing LN software to
+  manage [LN-Penalty][topic ln-penalty] payment channels within a
+  [channel factory][topic channel factories] using a software plugin.
+  The specification changes would allow the factory manager (e.g. a
+  Lighting service provider, LSP) to send messages to an LN node that
+  would be passed through to a local factory plugin.  Many factory
+  operations would be similar to [splicing][topic splicing] operations,
+  allowing the plugin to reuse a significant amount of code.  LN-Penalty
+  channel operations within a factory would be similar to [zero-conf
+  channels][topic zero-conf channels], so they could also reuse existing
+  code.
+
+  ZmnSCPxj's design is focused on SuperScalar-style factories (see
+  [Newsletter #327][news327 superscalar]) but would probably be
+  compatible with other factory styles (and possibly other multiparty
+  contract protocols).  Rene Pickhardt [replied][pickhardt plug] to ask
+  about additional specification changes that could allow channels
+  within factories to be [announced][topic channel announcements] but
+  ZmnSCPxj [said][zmnscpxj plug2] he deliberately didn't consider those
+  in his design in order to allow it the specification change to be
+  adopted as fast as possible.
+
+- **Signet activity report:** Anthony Towns [posted][towns signet] to
+  Delving Bitcoin a summary of activity on the default [signet][topic
+  signet] related to proposed soft forks available through [Bitcoin
+  Inquisition][bitcoin inquisition repo].  The post looks at use of
```

Comment:
```suggestion
  Inquisition][bitcoin inquisition repo].  The post looks at
```
Or, "the post looks at the use of  [SIGHASH_ANYPREVOUT][topic sighash_anyprevout], including tests ..."

(In other words, remove "usage")